dev-clean/benchmarks/klee_fs_lib/llsc.h

```diff
@@ -0,0 +1,28 @@
+#ifndef LLSC_HEADER_H
```

Comment:
maybe merge llsc.h/llsc.c with llsc_client.h?

Reply:
llsc.h will be included by multiple .c files, then they will be linked together. But llsc_client.h contains function definitions, so we can not include them in multiple files.